### PR TITLE
adds validators for URLs

### PR DESCRIPTION
**Pull request by niranjan92**
https://github.com/markbates/validate/issues/9

Adds url resource validators

### CODE REVIEW

File: validators/url_is_present_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		{"http://www.google.com", true},
+		{"http://google.com", true},
+		{"https://www.google.cOM", true},
+		{"https://golang.Org", true},
+		{"https://invalid#$%#$@.Org", false},
+	}
+	for _, test := range tests {
+		v := URLIsPresent{Name: "URL", Field: test.url}
+		errors := validate.NewErrors()
+		fmt.Println(errors)
```
Comment (markbates) [inline validators/url_is_present_test.go]: Can you remove this print statement?

File: validators/url_is_present.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	"github.com/markbates/validate"
+)
+
+type URLIsPresent struct {
+	Name    string
+	Field   string
+	Message string
+}
+
+// IsValid performs the validation based on the email regexp match.
```
Comment (markbates) [inline validators/url_is_present.go]: Not sure this documentation is correct here?

**Review by markbates — CHANGES_REQUESTED** [review]

**Comment by niranjan92** [comment]
@markbates updated the PR

**Comment by markbates** [comment]
Awesome! Thanks!

**Pull request merged**